shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -130,16 +141,79 @@ void PlatformChannel::HandleMethodCall(
     }
     SetPreferredOrientations(orientations);
     result->Success();
-  } else if (method == kSetApplicationSwitcherDescriptionMethod) {
-    result->NotImplemented();
-  } else if (method == kSetEnabledSystemUIModeMethod) {
-    result->NotImplemented();
-  } else if (method == kSetSystemUIOverlayStyleMethod) {
-    result->NotImplemented();
```

Comment:
Why don't you leave them on to-do list somewhere or comments?

Reply:
Sure. I added these items to the existing issue (https://github.com/flutter-tizen/engine/issues/178). Thank you for your suggestion!

Reply:
you're welcome